package.json

```diff
@@ -22,6 +22,7 @@
   "scripts": {
     "build:all": "yarn workspaces foreach run build",
     "test:all": "yarn workspaces foreach run test",
+    "slither:all": "cp -rf node_modules/@openzeppelin packages/contracts/dollar/node_modules && cp -rf node_modules/@openzeppelin packages/contracts/ubiquistick/node_modules && cp -rf node_modules/@uniswap packages/contracts/dollar/node_modules && cp -rf node_modules/solidity-linked-list packages/contracts/dollar/node_modules && yarn workspaces foreach run slither",
```

Comment:
Moving node_modules isn't a preferred way to make slither working because foundry conversion is in progress right now. It doesn't have node_modules at all. And then this is out of concept of workspace. we need to find a way without moving/copying openzeppelin.

Reply:
Worst case scenario I would recommend a symlink `ln -s`

Reply:
Okay, let me try symlink. Thanks for your directions